src_plugins/create/ppx_deriving_create.cppo.ml

```diff
@@ -1,11 +1,9 @@
-#include "../compat_macros.cppo"
-
-open Longident
+open Ppxlib
 open Location
 open Asttypes
 open Parsetree
 open Ast_helper
-open Ast_convenience
+open Ppx_deriving.Ast_convenience
```

Comment:
We now have to use our own redefinition of `Ast_convenience`, so plugin code has to be adapted slightly. (Note: *not* redefining `Ast_convenience` would mean cumbersome changes for third-party plugin authors.)

Reply:
The most notable change for this plugin is that essentially no (other) code change is required for plugin authors. (The patchset only get rids of now-useless IFDEFs.)